src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs

```diff
@@ -127,6 +128,8 @@ public MsgPackLogExporter(GenevaExporterOptions options)
             this.m_customFields = customFields;
         }
 
+        this.exporterStateForScopes = new ExporterStateForScopes() { CustomFields = this.m_customFields };
```

Comment:
nit: Just reading the diff, it doesn't seem like `ExporterStateForScopes` is needed. You could do something more or less like this...

```csharp
class MsgPackLogExporter
{
    private readonly ThreadLocal<SerializationDataForScopes> m_serializationData = new(() => null);

    private static readonly Action<LogRecordScope, MsgPackLogExporter> ProcessScopeForIndividualColumns = (scope, state) =>
    {
        var stateData = state.m_serializationData.Value;
        var customFields = state.m_customFields;

        // Do work
    }
}
```

Basically use `MsgPackLogExporter` as the state type.

Reply:
Great suggestion! Updated.